trio/_core/_run.py

```diff
@@ -61,6 +62,9 @@
 
 _r = random.Random()
 
+# Used to log exceptions in instruments
+LOGGER = logging.getLogger("trio.Runner.instrument")
```

Comment:
Minor nitpick: can we make this:
```
INSTRUMENT_LOGGER = logging.getLogger("trio.abc.Instrument")
```
Rationale: 
* More specific name since there's a ton of stuff in this file, some of which might potentially want to use logging some day, and most of it isn't related to instruments.
* `Runner` is a private class, so I'd rather not expose it as part of our logging names -- the string we pass here is effectively public API because people can filter on it etc. `trio.abc.Instrument` is a little odd looking -- let me know if you have a better idea -- but it's a public name and relevant so seems reasonable enough?

Reply:
I wasn't sure about giving the `LOGGER` a more specific name, that works great.

I also wasn't familiar enough with the API surface to pick a good logger name. `trio.abc.Instrument` works great.